src/main/java/uk/gov/companieshouse/web/pps/util/PenaltyUtilsService.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.companieshouse.web.pps.util;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.text.DecimalFormat;
+
+@Service
+public class PenaltyUtilsService {
```

Comment:
Think this class could be renamed back to `PenaltyUtils` and the annotation can just be `@Component`.
I'm wary of using Service here in the naming because this exists here in a new `util` package we have introduced for the sanctions work.

You'll notice that searching `@Service` in the project shows 5 others under a `service` package with interface and impl classes with the annotation which is a good pattern to follow.
Even if this util class is to be short lived, it would probably be worth tidying up the name / annotation now to keep it under the `util` package.

See https://github.com/search?q=repo%3Acompanieshouse%2Fpenalty-payment-web%20%40Service&type=code

Reply:
Fixed in latest commit